spec/rails_helper.rb

```diff
@@ -58,6 +58,7 @@
     DatabaseCleaner.cleaning do
       example.run
     end
+    clear_headers
   end
```

Comment:
Not really an issue (yet) since the helpers are small, but rather than add various methods to the main rspec helper, I prefer to use a module and then extend it here.  The other advantage of that is that you can include it in only the type of specs that need the methods:
https://relishapp.com/rspec/rspec-core/v/2-13/docs/helper-methods/define-helper-methods-in-a-module#include-a-module-in-only-some-example-groups